helpers/Cron.js

```diff
@@ -59,9 +59,10 @@ const healthCheck = () => {
           status: 'failed',
         };
         try {
-          await Fetch(`${scraper.baseUrl}/${scraper.version}/health`);
+          await Fetch(`${scraper.baseUrl}/health`);
```

Comment:
add back version to fetch url. `await Fetch(`${scraper.baseUrl}/${scraper.version}/health`);`